### ISSUE TITLE
On a multi-users setup, in the lock screen (CTRL+ALT+L)...

### ISSUE THREAD

**Issue by orbitalchicken** [linuxmint/cinnamon-screensaver#393]
```
 * Cinnamon-Screensaver version cinnamon-screensaver 5.0.6 and (probably some previous versions)
 * Distribution - (Mint 19.x and 20.x)
 ```

**Issue**
Keyboard navigation does not work to return to the main login screen.

**Steps to reproduce**
1) In the lock screen, when the blinking cursor is the password field, I press TAB, TAB to highlight the following icon: "go to the main login screen / show all users" (this step works correctly)
2) With the icon "go to the main login screen / show all users" highlighted, I press ENTER. It display "invalid password" (wrong).

**Expected behaviour**
Pressing TAB, TAB, ENTER, should go to the main login screen showing all users.
